apps/teams-test-app/e2e-test-data/authentication.json

```diff
@@ -19,19 +19,19 @@
       "expectedTestAppValue": "Success: {\"oid\":\"mockoid\",\"tid\":\"mocktid\",\"upn\":\"mockupn\",\"loginHint\":\"mockLoginHint\",\"displayName\":\"mockName\",\"dataResidency\":\"public\"}"
     },
     {
-      "title": "getUser API Call - Failure (Not full trust)",
+      "title": "getUser API Call - Failure (Not microsoft owned)",
       "version": ">2.25.0",
       "hostSdkVersion": {
         "web": "<=3.0.2"
       },
       "type": "callResponse",
       "platformsExcluded": ["iOS"],
       "boxSelector": "#box_getUser",
-      "testUrlParams": [["appDefOverrides", "{\"isFullTrustApp\": false}"]],
+      "testUrlParams": [["appDefOverrides", "{\"isFullTrustApp\": false, \"isMicrosoftOwned\": false}"]],
```

Comment:
Does `isFullTrustApp` still need to be here? I guess it would for backwards compatibility.

Reply:
@juanscr and I discussed this offline and determined that this is fine for now. I'm resolving this comment as a result.